pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -218,7 +216,7 @@ def _register_to_signal(self):
         signal.signal(signal.SIGUSR2, self._on_drain_signal)
 
     def _on_drain_signal(self, signal_number, frame):
-        self._logger.debug_with('Received signal, calling draining callback', signal=signal_number)
+        self._logger.debug_with('Received signal, calling draining callback', signal=signal_number.name)
```

Comment:
I think it should be as follows, but please verify:
```suggestion
        self._logger.debug_with('Received signal, calling draining callback', signal=signal.Signals(signal_number).name)
```

Reply:
@TomerShor yes, you're right